### DISCUSSION TITLE
How can I connect the datasets to train my model?

### DISCUSSION THREAD

**Discussion by AlphaDoraem** [huggingface/datasets#4209] [Q&A]
Hi,
I met a question when I train my [SimCSE] model on my own datasets. After "python train.py", there was an error:

ConnectionError: Couldn‘t reach https://raw.githubusercontent.com/huggingface/datasets/1.2.1/datasets/csv/csv.py

So I download the csv.py and put it in my dataset directory(datasets\csv\csv.py)

and run the code again. But I got no response but just one line of output:

"Using custom data configuration default".

What's the problem?  Please help me.

![image](https://user-images.githubusercontent.com/67501229/165005895-37a68da1-8be7-4e97-8b06-67881d496872.png)

![image](https://user-images.githubusercontent.com/67501229/165005767-f534e6f0-e5a5-4080-9a83-f6c4b53cc9b2.png)

**Comment by AlphaDoraem** [comment]
It seems that the program stops at this line of code: 

if extension == "csv":
        datasets = load_dataset(".\\datasets\\csv", data_files=data_files, cache_dir="./data/", delimiter="\t" if "tsv" in data_args.train_file else ",")

**Comment by AlphaDoraem** [comment]
new error:
![image](https://user-images.githubusercontent.com/67501229/165125308-67e82413-cc56-4e74-9644-1771e303ae04.png)
there is no response like: "Using custom data configuration default".

**Comment by mariosasko** [comment]
Hi! What version of `datasets` are you using? The `csv` module comes preinstalled with the lib in the newer releases, so please update your `datasets` installation with `pip install -U datasets`.
